README.md

```diff
@@ -57,10 +57,11 @@ To actually have the code serve your specific needs, you can modify the yaml fil
 | fetch_SED              | bool   | If true, get the SN SED from the OpenUniverse parquet files. If false, use a flat SED. May not be perfectly functional yet. TODO: see if this is improvable. |
 | make_initial_guess     | bool   | If true, the algorithm uses an average of the pixel values at each model point to set an initial guess for each model point. Slight improvement in certain cases but not pivotal. |
 | source_phot_ops        | bool   | If true, use photon shooting to generate the PSF for fitting the SN. This seemingly needs to be true for a quality fit.     |
+| flux_initial_guess | float | For making the initial guess, what flux should the SN be set to? This makes practically zero difference to the results, there just needs to be a choice made. |
```

Comment:
Can you provide better guidance to the user?
"This makes practically zero difference to the results, there just needs to be a choice made." doesn't really help.  Should this just be `1`?  Does it ever matter if it's not `1`?